config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: 506463145083.dkr.ecr.us-west-2.amazonaws.com/cwagent-operator-pre-release
+  newName: 355660395157.dkr.ecr.us-east-1.amazonaws.com/amazon-cloudwatch-agent-operator
```

Comment:
Is this intentional ? This seems like a personal account

Reply:
good catch, actually both of these images should be changed to use the public image since the previous image was also private